lib/check-minor-version.js

```diff
@@ -0,0 +1,19 @@
+var semver = require('semver')
+var getElectronVersion = require('electron-version')
+
+module.exports = function (app) {
+  if (!app || !app.path) return
```

Comment:
It looks like only the `path` is used from the `app` so maybe the path should be passed in instead.